spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -352,19 +352,6 @@
     end
   end
 
-  context 'user signs up with existing email' do
-    it 'sends an email to the existing user' do
-      existing_user = create(:user, :signed_up)
-
-      mailer = instance_double(ActionMailer::MessageDelivery)
-      expect(UserMailer).to receive(:signup_with_your_email).
-        with(existing_user.email).and_return(mailer)
-      expect(mailer).to receive(:deliver_later)
-
-      put :create, user: { email: existing_user.email }
-    end
-  end
-
```

Comment:
I removed this because we already have a feature spec that tests the same thing, as well as the text content on the redirect, which can't be tested in a controller spec.